### ISSUE TITLE
apt-get install sextractor does not place or creates the binary correctly

### ISSUE THREAD

**Issue by simontorres** [astromatic/sextractor#25]
I'm installing sextractor and scamp in a docker container based on debian and while scamp creates an executable `scamp`  sextractor does not. I also tried in a Linux Neon install which is based on Ubuntu.

Running `dpkg -L sextractor` shows.

```
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/sextractor
/usr/share/doc/sextractor/copyright
/usr/share/doc/sextractor/changelog.Debian.gz
```

Running `dpkg -L scamp` shows:

```
/.
/usr
/usr/bin
/usr/bin/scamp
/usr/share
/usr/share/doc
/usr/share/doc/scamp
/usr/share/doc/scamp/BUGS
/usr/share/doc/scamp/README.md
/usr/share/doc/scamp/changelog.Debian.gz
/usr/share/doc/scamp/changelog.gz
/usr/share/doc/scamp/copyright
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/scamp.1.gz
/usr/share/scamp
/usr/share/scamp/cfh12k.ahead
/usr/share/scamp/decam.ahead
/usr/share/scamp/fors2.ahead
/usr/share/scamp/megacam.ahead
/usr/share/scamp/minimo.ahead
/usr/share/scamp/mosaic2.ahead
/usr/share/scamp/omegacam.ahead
/usr/share/scamp/scamp.xsl
/usr/share/scamp/suprimecam.ahead
/usr/share/scamp/vircam.ahead
/usr/share/scamp/wfc.ahead
/usr/share/scamp/wfi.ahead
/usr/share/scamp/wircam.ahead
```
The versions installed are (from `apt-list`):
sextractor/stable,now 2.25.0+ds-3 all [installed]
scamp/stable,now 2.10.0-2 amd64 [installed]

Thanks in advance!.

**Comment by simontorres** [comment]
Even though this is not the expected solution I was able to install sextractor from github, this is what I added to my Dockerfile
`libgl1-mesa-glx sextractor scamp` where already there so maybe you don't need them.
```
FROM python:3.9
# included the base image here for reference.


# there are some other generic stuff here

# originally I just needed to add sextractor and scamp and would get a working sextractor
RUN apt-get update && apt-get install -y libgl1-mesa-glx sextractor scamp libatlas-base-dev libatlas3-base libfftw3-3 libfftw3-dev libtool autoconf

# basically the instructions from the INSTALL file in the root of this repo.
RUN mkdir -p /home/simon/sources/ &&  cd /home/simon/sources/ && git clone https://github.com/astromatic/sextractor.git && cd sextractor && sh ./autogen.sh && ./configure && make -j && make install

# and whatever else you need to do with your image creation here.

```

I hope someone can fix the deb package or if someone wants to teach me I can take care of it.

**Comment by alexw-im** [comment]
Thank you for this! I thought I was going crazy ;)

**Comment by ebertin** [comment]
Unfortunately I can't help you much with the versions that come with Ubuntu or Debian, which are packaged by other people. I do provide optimized versions of the packages (instructions [here](https://www.astromatic.net/repositories)). I am currently re-factoring the building process for those packages in my spare time (moving from INTEL icc to the INTEL icx compiler). It should be complete by mid-October I think.

**Comment by cmarmo** [comment]
For completeness, please note that sextractor has been renamed by debian maintainers as [source-extractor](https://packages.debian.org/stable/sextractor), so I guess you have to check your installation with
```
dpkg -L source-extractor
```

**Comment by YushengEdward** [comment]
> Even though this is not the expected solution I was able to install sextractor from github, this is what I added to my Dockerfile `libgl1-mesa-glx sextractor scamp` where already there so maybe you don't need them.
> 
> ```
> FROM python:3.9
> # included the base image here for reference.
> 
> 
> # there are some other generic stuff here
> 
> # originally I just needed to add sextractor and scamp and would get a working sextractor
> RUN apt-get update && apt-get install -y libgl1-mesa-glx sextractor scamp libatlas-base-dev libatlas3-base libfftw3-3 libfftw3-dev libtool autoconf
> 
> # basically the instructions from the INSTALL file in the root of this repo.
> RUN mkdir -p /home/simon/sources/ &&  cd /home/simon/sources/ && git clone https://github.com/astromatic/sextractor.git && cd sextractor && sh ./autogen.sh && ./configure && make -j && make install
> 
> # and whatever else you need to do with your image creation here.
> ```
> 
> I hope someone can fix the deb package or if someone wants to teach me I can take care of it.

Thanks for your anwser! I have solved my problem but i added sudo to make it work.

**Comment by rmathar** [comment]
The command on Ubuntu is
```
sudo apt-get install source-extractor
```
The issue can be closed as solved.
